lib/msal-angular/src/constants.ts

```diff
@@ -11,6 +11,7 @@ export const MSAL_GUARD_CONFIG = new InjectionToken<string>("MSAL_GUARD_CONFIG")
 
 export const MSAL_INTERCEPTOR_CONFIG = new InjectionToken<string>("MSAL_INTERCEPTOR_CONFIG");
 
-export const name = "@azure/msal-angular";
-
-export const version = "2.0.0-alpha.1";
+export const Constants = {
+    SKU: "@azure/msal-angular",
+    VERSION: "2.0.0-alpha.3"
```

Comment:
Lets try to solve pulling this in dynamically tomorrow.

Reply:
Addressed in #2915